### PR TITLE
[4.0] Array to php 5.4+ notation [Plugins 4: TwoFactorAuth and User]

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Pull Request for Improvement.

### Summary of Changes

Since 4.0 doesn't support php 5.3 anymore, use php 5.4+ array notation in TwoFactorAuth and User plugins.

### Testing Instructions

Simple code review.

### Documentation Changes Required

None.

### CODE REVIEW

File: plugins/user/profile/profile.php [hunk 1]
```diff
@@ -435,7 +435,7 @@ public function onUserAfterSave($data, $isNew, $result, $error)
 			$db->setQuery($query);
 			$usedOrdering = $db->loadColumn();
 
-			$tuples = array();
+			$tuples = [];
```
Comment (Quy) [inline plugins/user/profile/profile.php]: Align `=`

**Comment by frankmayer** [comment]
I have tested this item :white_check_mark: successfully on 646d71b92fd7a3f248e032f255e46dc3acb932ce<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16655">issues.joomla.org/tracker/joomla-cms/16655</a>.</sub>

File: plugins/user/profile/profile.php (hunk 1 above)
Comment (andrepereiradasilva) [inline plugins/user/profile/profile.php]: aligned

**Review by frankmayer — APPROVED** [review]

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c78e9d11e476fc3e576662318c08914c3ce02ad6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16655">issues.joomla.org/tracker/joomla-cms/16655</a>.</sub>

**Pull request closed without merging**